src/components/MarketWithDirectionLabel/MarketWithDirectionLabel.tsx

```diff
@@ -0,0 +1,28 @@
+import { t } from "@lingui/macro";
+import cx from "classnames";
+
+import TokenIcon from "components/TokenIcon/TokenIcon";
+
+export function MarketWithDirectionLabel({
+  indexName,
+  isLong,
+  tokenSymbol,
+  bordered,
+}: {
+  indexName: string;
```

Comment:
you mean change the name,
or pass whole marketInfo entity as a prop?

Reply:
whole marketInfo? Or you think it might be worse memo-wise?

Reply:
i think its just simpler 
its only two props instead of one. If there were multiple market props here -- then maybe. but now its only indexName and tokenSymbol